docs/src/pages/components/cdk-components/sidenav/Sidenav.jsx

```diff
@@ -29,17 +31,17 @@ function Sidenav() {
           ></DxcLink>
           .
         </p>
-        <DxcHeading text="DxcSidenav.Title" level={4} weight="bold" />
-        <p>The content will be showed as the main title in the sidenav.</p>
-        <DxcHeading text="DxcSidenav.Subtitle" level={4} weight="bold" />
-        <p>The content will be showed as a subtitle.</p>
-        <DxcHeading text="DxcSidenav.Link" level={4} weight="bold" />
+        <DxcHeading text="DxcSidenav.Title" level={4} weight="bold" margin={{ bottom: "small", top: "small" }} />
```

Comment:
I would add a short description for both Title and Subtitle (for example, 'Customized title for the sidenav.'). Also, we should add a subtitle Props, as it is done in the Link subcomponent.

![image](https://user-images.githubusercontent.com/44321109/152948816-e760c5e1-8b52-4f0a-9a36-e43f1c4f01af.png)